Monika After Story/game/script-story-events.rpy

```diff
@@ -656,9 +657,9 @@ label mas_random_ask:
             m 1eua "Great!"
             m "If you get tired of watching me talk about the same things over and over,{w} just open up the settings and uncheck 'Repeat Topics'."
             # TODO: this really should be a smug or wink face
-            m "That tells me when {cps=*2}you're bored of me{/cps}{nw}"
+            m 1tua "That tells me when {cps=*2} you're bored of me.{/cps}{nw}"
```

Comment:
I thought we decided that this shouldn't be a smug kind of expression. 

You have an extra space at the beginning of the `{cps}`.

You don't need a period at the end since it auto progresses and that isn't meant to be the end of that sentence.